Assets/GothicVR/Scripts/Manager/SkyManager.cs

```diff
@@ -114,6 +176,98 @@ private void Interpolate()
             DynamicGI.UpdateEnvironment();
         }
 
+        private void InitRainState()
+        {
+            // values taken from the original game
+            rainState.time = 0.187f; // 16:30
```

Comment:
Why not using DateTime as we use it for MinuteChange etc.?

Reply:
it is easier to keep track with how the original game handled skybox rendering including rain and night sky pfx